package.json

```diff
@@ -106,6 +105,13 @@
     "tape-catch": "~1.0.6",
     "testdouble": "^3.8.2",
     "testdouble-timers": "^0.1.1",
-    "tmp": "~0.0.33"
+    "tmp": "~0.0.33",
+    "ts-loader": "^7.0.5",
+    "ts-node": "^8.10.2",
+    "typedoc": "next",
+    "typedoc-plugin-markdown": "^2.3.1",
+    "typescript": "^3.9.5",
+    "webpack": "^4.43.0",
```

Comment:
Where is webpack used?

Reply:
for `npm run bundle` which creates `dist/bundle.js`